query/src/main/kotlin/com/linecorp/kotlinjdsl/querydsl/QueryDslImpl.kt

```diff
@@ -193,6 +218,13 @@ open class QueryDslImpl<T>(
         return getJoinClause()
     }
 
+    protected fun getSimpleAssociatedJoinClauseOnly(): SimpleAssociatedJoinClause {
+        return joins.filterIsInstance<SimpleAssociatedJoinSpec<*, *>>().let {
+            mustBe(it.size == joins.size) { "This query only support associate join" }
```

Comment:
-- english
DSL uses the method name associate, so it would be good to remove the `join` word.

-- korean
DSL에서는 assoicate라는 메소드 명을 사용하기 때문에 join 단어를 제거해도 좋을 것 같습니다.

Reply:
applied

-- korean
반영했습니다

Reply:
-- english
It is not applied yet

-- korean
아직 반영이 안 된 것 같습니다.

Reply:
I misunderstood and changed the method name
applied again.

-- korean
잘못 이해하고 메소드 이름을 바꿨네요
다시 반영했습니다.